.azure-pipelines/client.yml

```diff
@@ -124,9 +149,16 @@ jobs:
       summaryFileLocation: '$(Build.SourcesDirectory)/coverage.xml'
       reportDirectory: '$(Build.SourcesDirectory)/htmlcov'
 
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
+    displayName: 'Component Detection'
+    condition: and(eq(variables['os.name'],'Windows'),succeeded())
+
 - job: Test_Alpha_Python
   timeoutInMinutes: 90
   condition: eq(variables['long_running_tests'], True)
+
+
+- job: Special_Python_Distro_Tests
```

Comment:
Is this related to the CG change?

Reply:
No, this job is not related to CG support

Reply:
Why is this change part of this PR?  If it's unrelated to CG support it should ideally be in a separate PR.

Reply:
The job Special_Python_Distro_Tests is from an old commit and is no longer there in the master. This happened because the merge went wrong while resolving conflicts. But I pushed a new update, that has the correct changes that should be there.